api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Claimant.java

```diff
@@ -45,4 +45,7 @@
     @NotNull
     @OneToOne(cascade = CascadeType.ALL)
     private Address address;
+
+    @Column(name = "phone_number")
```

Comment:
Should we add validation here? We could add a regex pattern that enforces '+44' followed by 9 or 10 digits

Reply:
I was planning on adding that in the next PR as we'll need NotNull on it as well. Potentially may use https://github.com/google/libphonenumber as recommended by govUK, but a simple regex may suffice.

Reply:
We might want to be a bit more lenient on the phone number, there's a fair chance that some of the applicants might enter a non UK phone number.

Reply:
In the kick off I think we agreed that it had to be a UK number